pyproject.toml

```diff
@@ -31,16 +31,6 @@ classifiers = [
     "Topic :: Software Development :: Testing :: Unit",
     "Typing :: Typed",
 ]
-include = [
-    "CHANGELOG.md",
-    "docs/**/*.md",
-    "LICENSE",
-    "README.md",
-    "src/**/*.py",
-    "tests/**/*.py",
-    "docs/conf.py",
-    "docs/toctree.rst",
-]
```

Comment:
You _can_ include these only in the sdist:

```toml
include = [
  { path = "CHANGELOG.md", format = "sdist" },
  { path = "docs/**/*.md", format = "sdist" },
  { path = "LICENSE", format = "sdist" },
  { path = "README.md", format = "sdist" },
  { path = "tests/**/*.py", format = "sdist" },
  { path = "docs/conf.py", format = "sdist" },
  { path = "docs/toctree.rst", format = "sdist" },
]
```

Reply:
Exactly what I was looking for! Thank you!

Reply:
No problem :)